src/extensions/renderer-api/k8s-api.ts

```diff
@@ -29,3 +30,15 @@ export { Role, roleApi } from "../../renderer/api/endpoints";
 export { RoleBinding, roleBindingApi } from "../../renderer/api/endpoints";
 export { ClusterRole, clusterRoleApi } from "../../renderer/api/endpoints";
 export { ClusterRoleBinding, clusterRoleBindingApi } from "../../renderer/api/endpoints";
+
+export type KubeObjectStatus = {
```

Comment:
Does not feel right to have them here, everything else is exported from somewhere outside.

Reply:
Moved this types to separate file where those are exported from now